nautobot/core/api/urls.py

```diff
@@ -36,10 +34,8 @@
 ui_api_patterns = [
     # Lookup Expr
     path("core/", include((core_api_patterns, "core-api"))),
-    path("new-ui-ready-routes/", NewUIReadyRoutesAPIView.as_view(), name="new-ui-ready-routes"),
     path("get-menu/", GetMenuAPIView.as_view(), name="get-menu"),
     path("get-object-counts/", GetObjectCountsView.as_view(), name="get-object-counts"),
```

Comment:
I think `GetObjectCountsView` was also implemented specifically for the React UI

Reply:
Agreed; I'd left `GetMenuAPIView` and `GetObjectCountsView` in place for now because I was thinking that we could update the current nav-menu rendering and home-page population to be API-driven at some point even in the current UI paradigm, in which case these would become relevant again. Happy to rip them out for now if you think that'd be cleaner though.

Reply:
Nope that sounds like a good reason to leave them in